descriptor.md

```diff
@@ -34,9 +34,11 @@ The following fields contain the primary properties that constitute a Descriptor
   This property exists so that a client will have an expected size for the content before processing.
   If the length of the retrieved content does not match the specified length, the content SHOULD NOT be trusted.
 
-- **`urls`** *array*
+- **`urls`** *array of strings*
```

Comment:
Done in 8729ba4 → 876489a, and I've filed #447 with a handful of these “array of {type}” fixes.